### PR TITLE
Add Markdown linting and formatting

### PR DESCRIPTION
**Pull request by mondeja**
To follow a consistent style across all docs. Fixes two errors in third party integrations tables.

### CODE REVIEW

**Comment by LitoMore** [comment]
How about using the Prettier instead?

It's much easier for the developers to configure the formatter in the editor.

**Comment by uncenter** [comment]
Sorry, I'm looking for `"proseWrap": "never"` 😅

**Comment by mondeja** [comment]
Now is using `"proseWrap": "never"` 👍🏼

**Comment by uncenter** [comment]
You'll need to remove the changes from when you had it enabled earlier - prettier won't undo that for you, it'll preserve existing formatting like that.

**Comment by mondeja** [comment]
> You'll need to remove the changes from when you had it enabled earlier - prettier won't undo that for you, it'll preserve existing formatting like that.

`"proseWrap": "never"` wraps all paragraphs into one line (see https://prettier.io/docs/en/options.html#prose-wrap).  The rest of the changes are from Prettier or are intended for the Markdownlint checking.

**Review by LitoMore — APPROVED** [review]

File: .github/workflows/publish.yml
Comment (uncenter) [inline .github/workflows/publish.yml:1]: Is there a specific reason to make these multiline? Looks messy to me.

Comment (LitoMore) [reply .github/workflows/publish.yml:1]: It looks more like @mondeja's preference. Both formats are acceptable to me.

In my experience, if you use an editor on the command line and do not turn on line wrapping, the part that exceeds the screen will be difficult to read.

But see here: https://github.com/simple-icons/simple-icons/actions/runs/11991639440/job/33430520601. We have to expand the line to see the rest of the content.

For long commands with some not-that-important parameters, it is okay to use the line wrap. But here is only one version string. I think it's fine to write in one line.

Comment (mondeja) [reply .github/workflows/publish.yml:1]: Sorry, this was formatted by Prettier. I've reverted these changes.

**Pull request merged**